x-pack/plugins/search_playground/public/components/view_code/examples/dev_tools.tsx

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+
+import { EuiCodeBlock } from '@elastic/eui';
+import { useFormContext } from 'react-hook-form';
+import { ChatFormFields } from '../../../types';
+
+export const DevToolsCode: React.FC = () => {
+  const { getValues } = useFormContext();
+  const query = getValues(ChatFormFields.elasticsearchQuery) ?? '';
```

Comment:
Suggestion: Not for this PR but would be good to use the code convertor to transform this REST API call into the language versions like python / javascript. Would be good to create an issue for this.

Reply:
This would be really nice addition 👍